src/pyscipopt/scip.pxi

```diff
@@ -1883,50 +3293,99 @@ cdef class Model:
         return vars
 
     def getNVars(self, transformed=True):
-        """Retrieve number of variables in the problems.
-        
-        :param transformed: get transformed variables instead of original (Default value = True)
+        """
+        Retrieve number of variables in the problems.
+
+        Parameters
+        ----------
+        transformed : bool, optional
+            Get transformed variables instead of original (Default value = True)
+
+        Returns
+        -------
+        int
+
         """
         if transformed:
             return SCIPgetNVars(self._scip)
         else:
             return SCIPgetNOrigVars(self._scip)
 
     def getNIntVars(self):
-        """gets number of integer active problem variables"""
+        """
+        gets number of integer active problem variables.
+
+        Returns
+        -------
+        int
+
+        """
         return SCIPgetNIntVars(self._scip)
 
     def getNBinVars(self):
-        """gets number of binary active problem variables"""
+        """
+        Gets number of binary active problem variables.
+
+        Returns
+        -------
+        int
+
+        """
         return SCIPgetNBinVars(self._scip)
 
     def getNImplVars(self):
-        """gets number of implicit integer active problem variables"""
+        """
+        Gets number of implicit integer active problem variables.
+
+        Returns
+        -------
+        int
+
+        """
         return SCIPgetNImplVars(self._scip)
 
     def getNContVars(self):
-        """gets number of continuous active problem variables"""
+        """
+        Gets number of continuous active problem variables.
+
+        Returns
+        -------
+        int
+
+        """
         return SCIPgetNContVars(self._scip)
     
     def getVarDict(self):
-        """gets dictionary with variables names as keys and current variable values as items"""
+        """
+        Gets dictionary with variables names as keys and current variable values as items.
+
+        Returns
+        -------
+        dict of str to float
```

Comment:
Is there no better way to represent this? Like "Dict: str -> float" or "Dict[str]: float" perhaps?

Reply:
Not using the official numpy style format. The format you're using there is for type hints (inside the code itself)